ark/mibi/tiling_utils.py

```diff
@@ -75,28 +83,28 @@ def generate_region_info(region_params):
     return region_params_list
 
 
-def _read_tma_region_input(fov_tile_info, region_params):
-    """Reads input for TMAs from user and fov_tile_info
+def _read_tma_region_input(fov_list_info, region_params):
```

Comment:
Now that we're only using the automatically generated TMA to label the FOVs from the manually generated one, we don't need the rest of these user inputs, like moly QC points, fov size, etc. The only thing that matters is constructing the grid of the appropriate size.

Reply:
Ok I agree, it was weird needing to include FOV size. When the user specifies the number of FOVs along the x- and y-axes now, we can assume the FOVs will be equally spaced along both axes according to the top-left and bottom-right FOVs defined.